platform/provider.go

```diff
@@ -90,6 +90,8 @@ func NewProvider(logger boshlog.Logger, dirProvider boshdirs.Provider, statsColl
 		devicePathResolver = devicepathresolver.NewVirtioDevicePathResolver(idDevicePathResolver, mappedDevicePathResolver, logger)
 	case "scsi":
 		devicePathResolver = devicepathresolver.NewScsiDevicePathResolver(500*time.Millisecond, fs)
+	case "scsiid":
+		devicePathResolver = devicepathresolver.NewScsiIDDevicePathResolver(50000*time.Millisecond, fs, logger)
```

Comment:
sphere use scsi path resolver. Here there are something happened to update the scsi-id to match disk-id seeing from hypervisor